apps/vc-api/src/vc-api/vc-api.controller.ts

```diff
@@ -363,4 +365,23 @@ export class VcApiController {
 
     return result;
   }
+
+  /**
+   * TODO: Needs to have special authorization
+   * @param exchangeDefinitionDto
+   * @returns
+   */
+  @Post('/workflows')
+  @ApiOperation({
+    description:
+      'Allows the creation of a new exchange by providing the credential query and interaction endpoints\n' +
+      'A NON-STANDARD endpoint currently.\n\n' +
+      'Similar to https://gataca-io.github.io/vui-core/#/Presentations/post_api_v2_presentations'
```

Comment:
Minor documentation fix
```suggestion
      'Allows the creation of a new workflow\n' +
      'See https://w3c-ccg.github.io/vc-api/#create-workflow'
```